atlas/packages/app/src/shared/components/VideoPreview/VideoPreview.styles.tsx

```diff
@@ -38,6 +39,7 @@ export const CoverHoverOverlay = styled.div`
   align-items: center;
 `
 
+const PlayIcon = () => <Icon icon="play" />
```

Comment:
Something broke here, the video preview no longer shows the play icon on hover:
<img width="350" alt="Screenshot 2020-09-07 at 12 55 19 PM" src="https://user-images.githubusercontent.com/12646744/92380460-7fd6ba80-f109-11ea-8a3d-aa08584c8bf1.png">